.circleci/config.yml

```diff
@@ -31,13 +30,12 @@ jobs:
           # fallback to using the latest cache if no exact match is found
           - v1-dependencies-
 
-      - run: mvn dependency:go-offline
+      #- run: mvn dependency:go-offline -DexcludeGroupIds=com.google.cloud.tools.opensource
 
       - save_cache:
           paths:
             - ~/.m2
           key: v1-dependencies-{{ checksum "pom.xml" }}
         
       # run tests!
-      - run: mvn integration-test
-
+      - run: mvn integration-test -Dmaven.home=/opt/apache-maven
```

Comment:
I haven't had to set this property before.  Why was it necessary?

Reply:
The dashboard runs the dependency plugin which needs to know where maven is. Once we switch to our own library it will no longer be necessary.